static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js

```diff
@@ -299,13 +302,21 @@ const buildPageTargetting = (
     // This can be removed once we get sign-off from third parties who prefer to use appNexusPageTargeting.
     page.pageAdTargeting = pageTargeting;
 
-    return pageTargeting;
+    return Promise.resolve(pageTargeting);
 };
 
-const getPageTargeting = (): { [key: string]: mixed } => {
-    if (Object.keys(myPageTargetting).length !== 0) return myPageTargetting;
+const getPageTargeting = async (): Promise<PageTargettingLoose> => {
+    if (Object.keys(myPageTargetting).length !== 0)
+        return Promise.resolve(myPageTargetting);
```

Comment:
If this function is `async`, this could simply return the page targetting object, as `async` functions always return a Promise.

```suggestion
        return myPageTargetting;
```